### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by sameerapolsani**

### CODE REVIEW

File: javascript/javascript2/week3/fastestPresser/fastestPresser.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  margin: 5px;
+}
+
+.keyS_container {
+  border-right: 1px solid black;
+}
+
+.keyL_container {
+  border-left: 1px solid black;
+}
```
Comment (dinythomas89) [inline javascript/javascript2/week3/fastestPresser/fastestPresser.css:82]: When using higher values, for example margin:100px, use relative lengths

File: javascript/javascript2/week3/homeWork/hw.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
+//Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+function delay() {
+  let d = new Date();
+  let n = d.toLocaleTimeString();
```
Comment (dinythomas89) [inline javascript/javascript2/week3/homeWork/hw.js:4]: Try to use meaningful variable names

File: javascript/javascript2/week3/homeWork/hw.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+//And vice versa.
+function jokeCreator(shouldTellFunnyJoke, logFunnyJoke, logBadJoke) {
+  return shouldTellFunnyJoke;
+}
+function shouldTellFunnyJoke(joke) {
+  if (joke === true) {
+    logFunnyJoke();
+  } else {
+    logBadJoke();
+  }
```
Comment (dinythomas89) [inline javascript/javascript2/week3/homeWork/hw.js:92]: try ternary operator

**Review by dinythomas89 — APPROVED** [review]

File: javascript/javascript2/week3/fastestPresser/fastestPresser.css (hunk 1 above)
Comment (sameerapolsani) [reply javascript/javascript2/week3/fastestPresser/fastestPresser.css:82]: Thank you diny

**Pull request merged**